{"name": "test_check_redirect_n_count_times[3]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 == 3\n  +0\n  -3", "trace": "test_redirect.py:29: in test_check_redirect_n_count_times\n    assert len(r.history) == n\nE   assert 0 == 3\nE     +0\nE     -3\n        n          = 3\n        r          = <Response [404]>\n        url        = 'https://httpbin.org/redirect/3'"}, "attachments": [{"name": "log", "source": "722a088e-673b-45fb-96e0-057f8f061545-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "97eb98dc-ca7d-4f3a-8765-076fd09cf236-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "n", "value": "3"}], "start": 1596531097321, "stop": 1596531099411, "uuid": "efa2b7d6-6f20-4491-83d4-cee2f1403bed", "historyId": "0f2d130afb9b0a5a4f32ad13bed50dc7", "testCaseId": "f07b42c17c576f1548b8a61317a6569e", "fullName": "test_redirect#test_check_redirect_n_count_times", "labels": [{"name": "suite", "value": "test_redirect"}, {"name": "host", "value": "S340"}, {"name": "thread", "value": "32140-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_redirect"}]}